cl/scrapers/management/commands/cl_scrape_ia.py

```diff
@@ -0,0 +1,96 @@
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+import json
+import argparse
+import requests
+from internetarchive import get_files, search_items
+from cl.lib.command_utils import VerboseCommand, logger
+from django.conf import settings
+
+def download_from_internet_archive(options):
+    """
+    Download cases from internet archive via case law and add write them to
+    disk.
+
+    Requires a reporter abbreviation to identify cases to download.
+
+    Opitionally pass in a volume number to download that volume only.  If no
+    Volume number provided the code will cycle through the entire reporter
+    collection on IA.
+
+    :param options:
```

Comment:
Flesh this out, and maybe change the signature of the function to take a volumena nd reporter params?